docs/rules/no-unlocalized-strings.md

```diff
@@ -5,18 +5,93 @@ Ensures that all string literals, templates, and JSX text are wrapped using `<Tr
 > [!IMPORTANT]  
 > This rule may require TypeScript type information. Enable this feature by setting `{ useTsTypes: true }`.
 
+This rule is designed to **match all** JSXText, StringLiterals, and TmplLiterals, and then exclude some of them based on attributes, property names, variable names, and so on.
+
+The rule doesn’t come with built-in ignore settings because each project is unique and needs different configurations. You can use the following config as a starting point and then adjust it for your project:
+
+<!-- prettier-ignore -->
+```json5
+{
+  "no-unlocalized-strings": [
+    "error",
+    {
+      "useTsTypes": true,
+      "ignore": [
+        // Ignore strings that do not contain words
+        "^[^A-Za-z]+$",
```

Comment:
i think this could be built-in in the rule, but changed to support non latin languages (Arabic, Chinese, etc) for example 

`^(?!.*\p{L}).*$` using unicode property shortcut `p{L}` which is matching any letter in any language.